.gitlab-ci.yml

```diff
@@ -46,16 +46,24 @@ deploy_to_docker_registries:
   rules:
     - if: '$POPULATE_CACHE'
       when: never
+    # Wait 1 day to trigger the downstream job.
+    # This is a work-around since there isn't a way to trigger
+    # Gitlab from the Github workflow (build_deploy.yml:upload_pypi).
+    #
+    # The caveat here is that if there is a failure to build to PyPI
+    # and it isn't fixed in a day then this job will fail and images
+    # will not be published.
     - if: '$CI_COMMIT_TAG =~ /^v.*/'
-      when: on_success
+      when: delayed
+      start_in: 1 day
     - when: manual
       allow_failure: true
```

Comment:
What's the impact when this workflow fails? Who gets notified? Would library injection fail or would the library be installed from source?

Reply:
This is a good question. AFAIK we don't have good visibility of any of the gitlab jobs. We should look into how we can monitor the jobs.

Reply:
Looks like we have the results in CI app. I will set up alerting if the pipeline fails.